.editorconfig

```diff
@@ -3,7 +3,7 @@ root = true
 [*]
 charset = utf-8
 indent_style = space
-indent_size = 2
+indent_size = 4
```

Comment:
This contradicts js lint and the fact most files are indented by 2. We should either leave this at 2 or switch everything to 4.